bedrock/base/templates/macros.html

```diff
@@ -345,8 +353,8 @@ <h3>{{ _('Your download link was sent.') }}</h3>
     {{ download_firefox(dom_id=download_id, download_location=button_location) }}
   </div>
   <div class="show-fxa-supported-signed-out">
-    <a href="{{ accounts_url }}" class="button button-hollow button-light" data-button-name="Create account" data-link-type="button" data-cta-position="{{ button_location }}" id="{{ account_id }}">
-      {{ _('Create account') }}
+    <a href="{{ accounts_url }}" class="button {{ button_class }}" data-button-name="{{ button_text }}" data-link-type="button" data-cta-position="{{ button_location }}" id="{{ account_id }}">
```

Comment:
I think we should leave the data-button-name as "Create account". The current prod setup means that it's tracked as a create account button in GA no matter what locale it's in, I think we should maintain that.

Reply:
Ah good point.